### PR TITLE
SmartGameBooster.SmartGameBooster version 5.2.0

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27970)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
@mstfbot Please hold this pull till someone approves it

**Comment by BajajAditya123** [comment]
@OfficialEsco  Please approve my changes

File: manifests/s/SmartGameBooster/SmartGameBooster/5.2.0/SmartGameBooster.SmartGameBooster.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SmartGameBooster.SmartGameBooster
+PackageVersion: 5.2.0
+PackageLocale: en-US
+Publisher: Smart Game Booster
+PackageName: Smart Game Booster
+License: © Smart Game Booster. All rights reserved.
+ShortDescription: It booosts FPS for better gaming experience. It protects your gaming accounts.
```
Comment (ItzLevvie) [inline manifests/s/SmartGameBooster/SmartGameBooster/5.2.0/SmartGameBooster.SmartGameBooster.locale.en-US.yaml:10]:
```suggestion
ShortDescription: It boosts FPS for better gaming experience. It protects your gaming accounts.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by BajajAditya123** [comment]
Thank you for correcting me🙏🙏🙏

**Comment by BajajAditya123** [comment]
Can anyone review my changes??

**Pull request closed without merging**